domains/minhphan.json

```diff
@@ -4,6 +4,9 @@
     "email": "phan48788@gmail.com"
   },
   "record": {
-    "URL": "https://minhphan.tech"
+    "A": [
+            "75.2.60.5"
+        ],
+    "TXT" : "dh=78c7c89ac00d07ec34dd5e1d68289a3a877353dd"
```

Comment:
discord verification TXT goes in a separate domain `_discord.minhphan.is-a.dev`

Reply:
So i need change TXT to _discord.minhphan.is-a.dev right ?

Reply:
Oh sr i read the docs again sr